pyproject.toml

```diff
@@ -77,6 +77,7 @@ clustering = [
 forecasting = [
     "pmdarima>=1.8,!=1.8.1,<2.1",
     "prophet>=1.1,<1.2",
+    "skpro<2.1.0",
```

Comment:
Based on previous discussions, this must also have a lower bound.

Reply:
done